bukkit/pom.xml

```diff
@@ -376,7 +376,6 @@
             <groupId>io.netty</groupId>
             <artifactId>netty-transport</artifactId>
             <version>${nettyVersion}</version>
-            <scope>test</scope>
```

Comment:
Could you check if this one gets shaded into the plugin? It might should be provided if we only reference on it.

Reply:
Moved to & fixed in #876